scientific_library/tvb/simulator/plot/power_spectra_interactive.py

```diff
@@ -156,7 +157,7 @@ def configure(self):
         LOG.debug("time_series shape: %s" % str(self.time_series.data.shape))
         #TODO: if isinstance(self.time_series, TimeSeriesSurface) and self.first_n == -1: #LOG.error, return.
         self.data = self.time_series.data[:, :, :self.first_n, :]
-        self.period = self.time_series.sample_period
+        self.period = self.time_series.sample_period/1000  #JC: convert sampling period of time_series.data in second
```

Comment:
Is this generic enough ?
Could we maybe wrap this line with a more detailed condition in which we consider `time_series.sample_period_unit` property?
Or use `1/self.time_series.sample_rate` ?

Reply:
Hi Lia,
Yes,  we could directly use `self.period = 1/self.time_series.sample_rate`.
I was also thinking that we could specify the unit of the converted data: `self.period_unit='s'`